src/token_list.py

```diff
@@ -4,7 +4,7 @@
 import requests
 
 # pylint: disable=line-too-long
-ALLOWED_TOKEN_LIST_URL = "https://raw.githubusercontent.com/gnosis/cow-dex-solver/main/data/token_list_for_buffer_trading.json"
+ALLOWED_TOKEN_LIST_URL = "https://token-list.cow.eth.link/"
```

Comment:
This link sometimes just doesn't work. Its a bad connection. I have also tried locally with `http` instead of `https` does anyone think this would make a difference in terms of reliability?

Reply:
`eth.link` URLs aren't always reliable. I think we've had issues with it in the past.

I'm not sure what a better alternative would be except for hosting it ourselves on `token-list.cow.fi`.

Reply:
I would be very much in favour of having a reliable host.

Reply:
FUN FACT: Using `.limo` instead of `.link` is apparently much more reliable.